src/js/components/Anchor/StyledAnchor.js

```diff
@@ -17,9 +17,11 @@ const sizeStyle = (props) => {
   if (props.size) {
     const size = props.size || 'medium';
     const data = props.theme.text[size];
+    const gap = props.theme.anchor?.size?.[size]?.gap;
```

Comment:
If we are passing `gap` to the Box component in Anchor.js why do we also need to include it here?

Reply:
yeah good call